editoast/src/views/infra/edition.rs

```diff
@@ -39,7 +39,13 @@ async fn edit<'a>(
         })
         .await?;
 
-    chartos::invalidate_zone(infra, chartos_config, &invalid_zone).await;
+    chartos::invalidate_zone(
```

Comment:
Can you add a TODO and/or create an issue about invalidate only layers that have been updated.

Reply:
Yes : https://github.com/DGEXSolutions/osrd/issues/2955